tools/civic/civic_qrt.py

```diff
@@ -21,24 +27,37 @@ def civic_reader():
     parser = argparse.ArgumentParser()
     parser.add_argument("-i", "--input_file", help="input file (assumed to be tab separated MAF format); defaults to stdin", default="-" )
     parser.add_argument("-o", "--output_file", help="output file; defaults to stdout", default="-" )
-    parser.add_argument("-c", "--host", help="url of reference civic host (default: %(default)s)", default="civic.genome.wustl.edu"  )
     parser.add_argument("-col", "--entrez_id_column", help="column in the input file that contains the entrez id. Defaults to 1 (0 based)", default=1,  type=int )
     args = parser.parse_args()
 
+    # Using the tool wrappers path to access the list of civic servers available
+    civic_wrapper_dir = os.path.dirname(os.path.realpath(__file__))
+    civic_servers_file = civic_wrapper_dir + "/" + "civic_servers.txt"
+    with open(civic_servers_file) as f:
+      for line in f:
+        proxy_handler()
+        try: 
+            response = urllib2.urlopen('http://' + line)
+        except: 
+            print line, 'is down!'
+        else: 
+            print line, 'is up!'
+            host = line.rstrip()
+

```

Comment:
I felt the same, and i wanted to run that idea by you today. Let's discuss when i am in campus.